app/controllers/admin/items_controller.rb

```diff
@@ -41,6 +41,6 @@ def item_params
   end
 
   def require_admin
-    render file: "/public/404" unless current_admin?
+    render file: "/public/404" unless current_user.current_admin?
```

Comment:
This method is also defined on the application controller as well. Is this intended? and if so, why?